install/requirements.txt

```diff
@@ -9,6 +9,10 @@ gguf
 # Tiktoken tokenizer for Llama 3 and other advanced models
 tiktoken
 
+# Tokenizers and jinja2 for other non-llama models that use HF tokenizers
```

Comment:
I added these here, but did not add `pytest` (yet). I think there's a pending conversation about introducing optional dependency sets, so it would make sense to add a `test` or `dev` set at that point, but I didn't want to accidentally carry `pytest` along as a runtime dependency.